src/components/Header/index.tsx

```diff
@@ -38,11 +33,24 @@ const Header = () => {
     const signIn = async () => {
         await authService.signIn();
         session.setAuthenticated(true);
+        setIsAuthenticated(true);
+    };
+
+    const signOut = async () => {
+        await authService.signOut();
+        session.setAuthenticated(false);
+        setIsAuthenticated(false);
+        // Client-side cleanup
+        dataApiEndpoint.remove();
+        dataApiClientId.remove();
+        dataApiTenantId.remove();
```

Comment:
why do we need to clean-up these?

Reply:
To remove keys from local storage. This is a good thing to do.

Reply:
Good point. Then yes, need to understand if we need to remove the keys upon sign out. Probably not.